include/wabt/ir.h

```diff
@@ -120,6 +121,7 @@ struct Const {
       case Type::F64: return 2;
       default: WABT_UNREACHABLE;
     }
+    // clang-format on
```

Comment:
you can use `git clang-format` (after `git add` but before `git commit`) - it formats only modified code.

Reply:
Yeah, I'm aware.  It just happened by accident and I was hoping to avoid this in the future.

Reply:
I removed the clang-format pragmas again.